besu/src/main/java/org/hyperledger/besu/controller/MainnetBesuControllerBuilder.java

```diff
@@ -48,6 +50,12 @@ protected MiningCoordinator createMiningCoordinator(
       final SyncState syncState,
       final EthProtocolManager ethProtocolManager) {
     final ExecutorService minerThreadPool = Executors.newCachedThreadPool();
+    Vertx vertx = Vertx.vertx();
```

Comment:
We have a shared vertx instance already created in Besu Command and passed around via RunnerBuilder.  Also, the StratumServer should be constructed in RunnderBuilder.build() method.  The vertx instance is a field of that builder.

Reply:
OK, I'd like to wait for #104 to go in first before I make that change, unless instructed otherwise.

Reply:
#104 is merged, so make the changes and I'll give it a full review then.

Does this include the other two PRs that were proposed last week?

Reply:
yes, this includes both PRs.